core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceSpec.scala

```diff
@@ -1334,13 +1335,19 @@ class WorkspaceServiceSpec extends AnyFlatSpec with ScalatestRouteTest with Matc
     // the "oldBillingAccount" value
     verify(services.gcsDAO).setBillingAccountName(
       any[GoogleProjectId],
-      ArgumentMatchers.eq(billingProject.billingAccount.get)
+      ArgumentMatchers.eq(billingProject.billingAccount.get),
+      any[OpenCensusSpan]
     )
   }
 
   it should "fail to create a database object when GoogleServicesDAO throws an exception when updating billing account" in withTestDataServices { services =>
-    when(services.gcsDAO.setBillingAccountName(GoogleProjectId("project-from-buffer"), RawlsBillingAccountName("fakeBillingAcct")))
-      .thenReturn(Future.failed(new Exception("Fake error from Google")))
+    doReturn(Future.failed(new Exception("failed")), null)
```

Comment:
We are spying on a real object here, and the `services.gcsDAO.setBillingAccountName` invocation in the `when()` of the previous approach was invoking the real method under the hood. So, we need to use this `doReturn` approach https://www.javadoc.io/doc/org.mockito/mockito-core/1.10.19/org/mockito/Mockito.html#13

Reply:
Nit: I'm assuming the change from `Fake error from Google` to `failed` intentional/negligible?